components/google-cloud/google_cloud_pipeline_components/experimental/custom_job/custom_job_sample.ipynb

```diff
@@ -0,0 +1,348 @@
+    "outputs:\n",
+    "- {name: output_value, type: String, description: 'Represents an output paramter.'}\n",
+    "implementation:\n",
+    "  container:\n",
+    "    image: google/cloud-sdk:latest\n",
+    "    command:\n",
+    "    - sh\n",
+    "    - -c\n",
+    "    - |\n",
+    "      set -e -x\n",
+    "      echo \"$0, this is an output parameter\" | gsutil cp - \"$1\"\n",
+    "    - {inputValue: input_text}\n",
+    "    - {outputPath: output_value}\n",
+    "\"\"\"\n",
+    ")"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Convert the component to a Vertex AI Custom Job"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "# This is a temporary work around for v0.1.4 to point to the correct base image for custom_job\n",
+    "gcpc.experimental.custom_job.custom_job._DEFAULT_CUSTOM_JOB_CONTAINER_IMAGE = 'gcr.io/ml-pipeline/gcp-launcher:0.1.4'\n",
+    "\n",
+    "custom_job_producer_op= gcpc.experimental.custom_job.custom_job.run_as_vertex_ai_custom_job(producer_op)"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Define the pipeline:"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "@dsl.pipeline(pipeline_root='', name='custom-job-sample-pipeline')\n",
+    "def pipeline(text: str = 'message'):\n",
+    "    custom_producer_task = custom_job_producer_op(input_text=text, gcp_project=PROJECT_ID ,gcp_region=REGION)"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Compile and run the pipeline\n",
+    "\n",
+    "Compile the pipeline:"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "from kfp.v2 import compiler\n",
+    "\n",
+    "compiler.Compiler().compile(\n",
+    "    pipeline_func=pipeline, package_path=\"simple_custom_job_spec.json\"\n",
+    ")"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "The pipeline compilation generates the `simple_custom_job_spec.json` job spec file.\n",
+    "\n",
+    "Next, instantiate an API client object and run the defined pipeline like this: "
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "from kfp.v2.google.client import AIPlatformClient\n",
+    "\n",
+    "api_client = AIPlatformClient(project_id=PROJECT_ID, region=REGION)\n",
+    "\n",
+    "response = api_client.create_run_from_job_spec(\n",
+    "          job_spec_path='simple_custom_job_spec.json',\n",
+    "          pipeline_root=PIPELINE_ROOT,  enable_caching=False\n",
+    "          )"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Cleaning up\n",
```

Comment:
consider remove anything not specific to this component from here, or remove this entirely if nothing specific to custom job.